use-cases/point-of-sale.md

```diff
@@ -1,70 +1,112 @@
+You can post charges to customer profiles with the classification of `Paymaster` regardless of whether it is attached to a checked-in reservation. The customer classification, `Cashlist` is used by properties to not allow charges sent to their bill. This is also commonly known as "No Post" in other PMS products.
+
+Note that having a `Cashlist` customer [classification](../operations/customers.md#customer-classification) does not automatically prevent Orders from being posted to the customer's billing. The POS system can decide whether to recognise this classification on a customer profile and prevent users from sending room charges from the POS system.
 
 *Note: Room numbers of some hotels consist of numbers, letters, and other characters.*
 
-### Charging in-house customers
+### Charging in-house customers or Paymaster accounts
 
-Once the customer profile to be charged is identified, the items can be posted onto their bill using the [Add order](../operations/services.md#add-order) operation. The order needs to be sent with its full name (e.g. Caesar salad, Beer, etc.) and not just “Item". The `AccountingCategoryId` will need to be used per item to allow for correct reporting for accounting systems.
+Once the customer profile to be charged is identified, the items can be posted onto their bill using the [Add order](../operations/services.md#add-order) operation. The order needs to be sent with its full name (e.g. Caesar salad, Beer, etc.) and not just “Item". The `AccountingCategoryId` will need to be used per item to allow for correct reporting for accounting systems. You can make use of `Notes` to record the associated ticket number from the POS system.
 
 ### Split payments
 
-If the POS supports split payments, e.g. one salad divided between two people, it must be sent to Mews as separate transactions with the item count rounded up to the nearest whole number.
+If the POS supports split payments, e.g. one salad divided between two people, it must be sent to Mews as separate transactions with the product `Count` or item `UnitCount` rounded up to the nearest whole number. 
+
+Example: One salad of €10.00 divided between two people are sent as two separate Orders, each with 1 salad at a price of €5.00.
 
 ### Rebates
 
-Rebates (or cancelled items) will need to be allowed by the property. The [Add order](../operations/services.md#add-order) endpoint, with the items to be cancelled, should be sent through with negative values. This applies to Outlet items as well.
+Rebates (or cancelled items) will need to be allowed by the property and can be applied to both Orders and Outlet bills. The POS system should use the applicable endpoint - [add order](../operations/services.md#add-order) or [Add outlet bill](../operations/finance.md#add-outlet-bills) - to post a rebate or cancelled item.
```

Comment:
there is lowercase add order but uppercase add outlet bill

Reply:
and others are upercase please unify it

Reply:
updated